README.md

```diff
@@ -229,7 +234,7 @@ var app = http.createServer(function onRequest (req, res) {
 }).listen(3000)
 ```
 
-## License 
```

Comment:
This clean-up is done automatically by Atom. Correct markdown don't accept blanks at the end of the line, except two blank spaces to force to do backline.